### PR TITLE
jj

### PR DESCRIPTION
**Pull request by Jarek901**
[](https://jitpack.io/#Jarek901/jitpack.io)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,8 +201,8 @@ Or, if you prefer the flat-squared style:
 ```
 https://jitpack.io/v/User/Repo.svg?style=flat-square
```
Comment (Jarek901) [inline README.md]: [[]

Comment (Jarek901) [inline README.md]: [[]

File: README.md [hunk 2]
```diff
@@ -201,8 +201,8 @@ Or, if you prefer the flat-squared style:
...
 ```
+[[![](https://jitpack.io/v/Jarek901/jitpack.io.svg)]
 
-[![Release](https://jitpack.io/v/jitpack/maven-simple.svg?style=flat-square)](https://jitpack.io/#jitpack/maven-simple)
```
Comment (Jarek901) [inline README.md]: [[]

**Comment by yb801925** [comment]
ok

**Pull request closed without merging**